dev/release/02-source-test.rb

```diff
@@ -84,6 +84,7 @@ def test_csharp_git_commit_information
   def test_python_version
     source
     Dir.chdir("#{@tag_name_no_rc}/python") do
+      sh("pip", "install", "numpy")
```

Comment:
We would rather not need to do this explicitly. Either remove the mandatory NumPy dependency (see other comment), or arrange to use `pyproject.toml` so that the dependency is installed transparently.

cc @jorisvandenbossche

Reply:
Yes, running sdist should in theory not need numpy I think, and is probably solved by moving the import inline

Reply:
@pitrou @jorisvandenbossche Thanks so much for reviewing! It has been updated.